optimism.tokenlist.json

```diff
@@ -1067,6 +1067,28 @@
       "extensions": {
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
+    },
+    {
+      "chainId": 1,
+      "address": "0x69af81e73a73b40adf4f3d4223cd9b1ece623074",
+      "name": "Mask Network",
+      "symbol": "MASK",
+      "decimals": 18,
+      "logoURI": "https://etherscan.io/token/images/mask_32.png",
```

Comment:
Can you add a 256x256 logo to the `/logos` folder in the repo pls and link to that instead

Reply:
Yeah

Reply:
Thanks but the logo is too big and won't display properly. Can you drop the size to something closer to 256x256?

Reply:
@developerfred can you update the logo please so we can merge

Reply:
@elenadimitrova I will compress it now.

Reply:
done @elenadimitrova

Reply:
This looks like a partial image, can you double check 
![Screenshot 2022-01-17 at 14 21 26](https://user-images.githubusercontent.com/703848/149768660-47fb5124-0e79-43c2-aa4b-c45bfa3dbcf8.png)